examples_tf2_py/examples_tf2_py/async_waits_for_transform.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import math
```

Comment:
It doesn't seem like we are actually using the math library here.  Please remove.

Reply:
Added linter tests in e0479dc3d7401b8e2607352954d08a5e0e59c150 and fixed linter errors in 48ca46e4c1a7d81d13799d725a0fbdeafbbb7cac